components/recruitment.py

```diff
@@ -10,6 +10,11 @@
 def get_recruit_embed(user_id: int, bot: RecruitBot) -> Optional[Tuple[discord.Embed, RecruitView]]:
     user = bot.rusers.get(user_id)
 
+    # this will never happen because we validate that the user exists in the command,
+    # but my IDE is complaining without it
+    if not user:
+        return None
```

Comment:
Don't return None, that'll freak out the IDE more.  Just raise some BS exception.

Reply:
Yeah we could change that in future, I think this is functional for now though